SNIClient.py

```diff
@@ -684,6 +684,8 @@ async def main() -> None:
         logging.info(f"Wrote rom file to {romfile}")
         if args.diff_file.endswith(".apsoe"):
             import webbrowser
+            async_start(run_game(romfile))
+            await _snes_connect(SNIContext(args.snes, args.connect, args.password), args.snes)
```

Comment:
awaiting the `while True` inside `_snes_connect` is not a good idea. If either SNI is missing or the emulator is not correctly configured, it'll not show the website and instead hang there with no user feedback.

Reply:
![Screenshot_20](https://github.com/ArchipelagoMW/Archipelago/assets/13184667/6ebb5c70-98d6-444e-b8e6-383126d57bba)